### PR TITLE
orchis-theme: 2022-10-19 -> 2023-02-26

### PR DESCRIPTION
**Pull request by fufexan**
###### Description of changes

Bump version, add tweaks, simplify assert.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/themes/orchis-theme/default.nix [hunk 1]
```diff
@@ -6,29 +6,33 @@
...
+, tweaks ? [ ] # can be "solid" "compact" "black" "primary" "macos" "submenu" "nord|dracula"
 , withWallpapers ? false
 }:
 
 let
-  validTweaks = [ "solid" "compact" "black" "primary" ];
-  unknownTweaks = lib.subtractLists validTweaks tweaks;
+  pname = "orchis-theme";
+
+  validTweaks = [ "solid" "compact" "black" "primary" "macos" "submenu" "nord|dracula" ];
```
Comment (kilianar) [inline pkgs/data/themes/orchis-theme/default.nix:16]: This list should contain both `nord` and `dracula`. The mutual exclusivity is ensured by `nordOrBlack`.

Comment (fufexan) [reply pkgs/data/themes/orchis-theme/default.nix:16]: You're right. I was distracted and didn't check the list when I copied it over from the tweaks comment.

File: pkgs/data/themes/orchis-theme/default.nix [hunk 2]
```diff
@@ -6,29 +6,33 @@
...
 }:
 
 let
-  validTweaks = [ "solid" "compact" "black" "primary" ];
-  unknownTweaks = lib.subtractLists validTweaks tweaks;
+  pname = "orchis-theme";
+
+  validTweaks = [ "solid" "compact" "black" "primary" "macos" "submenu" "nord" "dracula" ];
+
+  nordOrBlack = with builtins; lib.assertMsg (!(elem "nord" tweaks) || !(elem "dracula" tweaks)) ''
```
[the remark this answers is not in the document]
Comment (fufexan) [reply pkgs/data/themes/orchis-theme/default.nix:18]: Yeah, another typo of mine. Also, what do you suggest I do about the xor?

[the remark this answers is not in the document]
Comment (kilianar) [reply pkgs/data/themes/orchis-theme/default.nix]: Maybe something like `nordXorDracula`?

**Review by kilianar — APPROVED** [review]
It looks good to me now. However, there is still a nitpick regarding the naming of nordOrDracular, but I don't have a strong opinion on the matter.

**Comment by fufexan** [comment]
Well usually we don't say xor in English, so that's why I used or. Some user may not even know what an xor is.

**Comment by kilianar** [comment]
This seems to be an instance of the ambiguity inherent in natural language expressions. I am content with either version :)

**Comment by kilianar** [comment]
Just wanted to give you a heads up that the version [2023-02-26](https://github.com/vinceliuice/Orchis-theme/releases/tag/2023-02-26) has just been released.

**Comment by fufexan** [comment]
> Just wanted to give you a heads up that the version [2023-02-26](https://github.com/vinceliuice/Orchis-theme/releases/tag/2023-02-26) has just been released.

Thanks for the heads up!
I also renamed the `Or` to `Xor`.

**Review by kilianar — APPROVED** [review]
Result of `nixpkgs-review pr 210734` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>orchis-theme</li>
  </ul>
</details>

**Pull request merged**